modules/nf-core/picard/markduplicates/picard-markduplicates.diff

```diff
@@ -0,0 +1,25 @@
+Changes in module 'nf-core/picard/markduplicates'
```

Comment:
Why do we need this patch? Can you just use the [older version](https://github.com/nf-core/modules/blob/20b0918591d4ba20047d7e13e5094bcceba81447/modules/nf-core/picard/markduplicates/main.nf) of the module if the new one isn't compatible somehow?

Reply:
no, because I want the stub

Reply:
new updated picard module have stub, old one not, but new one has some issues with the tests due to some read group thingy...

Reply:
Ahh, gotcha